test/integration/node-specific/client_close.test.ts

```diff
@@ -155,8 +469,31 @@ describe.skip('MongoClient.close() Integration', () => {
 
     describe('SrvPoller', () => {
       describe('Node.js resource: Timer', () => {
+        // requires an srv environment that can transition to sharded
+        const metadata: MongoDBMetadataUI = {
+          requires: {
+            predicate: () =>
+              process.env.ATLAS_SRV_REPL ? 'Skipped: this test requires an SRV environment' : true
+          }
+        };
+
         describe('after SRVPoller is created', () => {
-          it.skip('timers are cleaned up by client.close()', async () => {});
+          it('timers are cleaned up by client.close()', metadata, async () => {
+            const run = async function ({ MongoClient, uri, expect, getTimerCount }) {
+              const client = new MongoClient(uri);
+              await client.connect();
+              const description = client.topology.s.description;
+              // simulate transition to sharded
+              client.topology.emit('topologyDescriptionChanged', description, {
+                ...description,
+                type: 'Sharded'
+              });
+              expect(client.topology.s.srvPoller._timeout).to.exist;
```

Comment:
Can you try using the srv records in the initial seedlist spec: https://github.com/mongodb/specifications/tree/master/source/initial-dns-seedlist-discovery/tests#test-setup Those should work then we don't have to get an atlas test uri in here somehow

Reply:
I tried the set-up, seems like we don't have the [SRV environment testing case set-up ](https://github.com/mongodb/node-mongodb-native/blob/41b066b2b286879085b05a58c0486aa1d24303bc/test/integration/initial-dns-seedlist-discovery/initial_dns_seedlist_discovery.spec.test.ts#L21)(we're not running SRV spec tests as of now).

We could set it up so this test case is used for more testing envs, but given that we want to wrap up the project by tomorrow, maybe we leave setting up an SRV testing environment to a later time?